_config.yml

```diff
@@ -407,16 +407,15 @@ valine:
 
 # Gitment
 # Introduction: https://imsun.net/posts/gitment-introduction/
-# You can get your Github ID from https://api.github.com/users/<Github username>
 gitment:
   enable: false
   mint: true # RECOMMEND, A mint on Gitment, to support count, language and proxy_gateway
   count: true # Show comments count in post meta area
   lazy: false # Comments lazy loading with a button
   cleanly: false # Hide 'Powered by ...' on footer, and more
   language: # Force language, or auto switch by theme
-  github_user: # MUST HAVE, Your Github ID
-  github_repo: # MUST HAVE, The repo you use to store Gitment comments
+  github_user: # MUST HAVE, Your Github Username
```

Comment:
This option needs a **GitHub Username** in fact.

But the description above leads to the `id` which return by the API.

Therefore, I remove the comment near the top and change the `ID` to `Username` to make it more clear.

Reply:
> the description above leads 

and 

> the comment near the top

means this comment 

```
# You can get your Github ID from https://api.github.com/users/<Github username>
```